### PR TITLE
Metadata: add some version var

### PR DESCRIPTION
**Pull request by stydxm**

### CODE REVIEW

File: CH32V103/FreeRTOS/README_pio.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 sys: reertos
```
Comment (KevinMX) [inline CH32V103/FreeRTOS/README_pio.md:2]: Wait what?
```suggestion
sys: freertos
```
Also, _zh?

File: PIC64GX/Ubuntu/README_LTS.md
Comment (KevinMX) [inline PIC64GX/Ubuntu/README_LTS.md:1]: Alwo _zh?

**Review by KevinMX — CHANGES_REQUESTED** [review]

File: CH32V103/FreeRTOS/README_pio.md (hunk 1 above)
Comment (stydxm) [reply CH32V103/FreeRTOS/README_pio.md:2]: This is an existing problem. Sorry I didn't notice it.

File: PIC64GX/Ubuntu/README_LTS.md
Comment (stydxm) [reply PIC64GX/Ubuntu/README_LTS.md:1]: seems no metadata exists in all files with "_zh" suffix

Comment (KevinMX) [reply PIC64GX/Ubuntu/README_LTS.md:1]: > seems no metadata exists in all files with "_zh" suffix

Missed that, never mind

**Review by KevinMX — APPROVED** [review]

**Pull request merged**